app/models/event.rb

```diff
@@ -0,0 +1,20 @@
+class Event < ApplicationRecord
+  belongs_to :creator, class_name: User.name, foreign_key: 'user_id'
```

Comment:
#### Associations

##### User as event creator

* [ ] Event model `belongs_to :creator, class_name: "User"` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

- No need to specify the foreign_key and the class_name calling the User.name to be able to use this association you have to create a migration that adds the creator_id attribute to the Events table